deployment-examples/aws/k8s/README.md

```diff
@@ -34,19 +31,23 @@ Once all the prerequisites have been installed and created, update the repositor
 
 For any changes for cronjob, configure the properties in [values.yaml](./values.yaml).
 
-## Create release.
+## Creating AWS cluster (optional)
+
+Configure all the required environment variables in the bash script [`create_aws_cluster`](create_aws_cluster.sh). After that, execute the script file.
```

Comment:
In the part `### Environment variables` above you wrote this:
```
update the repository secrets for the following fields in the repository.
```
I don't understand that sentence. Please update that sentence. For example here you have `Configure all the required environment variables in the bash script create_aws_cluster` which is clear.